types/index.d.ts

```diff
@@ -3,7 +3,7 @@ import { AnnotationPluginOptions, BoxAnnotationOptions, EllipseAnnotationOptions
 
 declare module 'chart.js' {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  interface PluginOptionsByType<TType extends ChartType = ChartType> {
+  interface PluginOptionsByType<TType extends ChartType> {
```

Comment:
I think this is now ready to be merged, since this is the way it is noted in the latest chart.js version:  
https://github.com/chartjs/Chart.js/blob/dd554e0d87fc525c2a4efcecead9cc3c280936f4/src/types/index.d.ts#L2936

Reply:
switched ready for review